src/firebolt/service/instance_type.py

```diff
@@ -41,6 +43,27 @@ def instance_types_by_name(self) -> Dict[InstanceTypeLookup, InstanceType]:
             for i in self.instance_types
         }
 
+    def cheapest_instance_in_region(self, region: Region) -> InstanceType:
+        # Get only awailable instances in region
+        response = self.client.get(
+            url=INSTANCE_TYPES_URL,
+            params={"page.first": 5000, "filter.id_region_id_eq": region.key.region_id},
+        )
+        instance_types = [
+            InstanceType.parse_obj(i["node"]) for i in response.json()["edges"]
+        ]
+        # Filter out instances without storage
+        instance_list = [
+            i
+            for i in instance_types
+            if i.storage_size_bytes and i.storage_size_bytes != "0"
+        ]
+        cheapest = min(
+            instance_list,
+            key=lambda x: x.price_per_hour_cents if x.price_per_hour_cents else maxsize,
```

Comment:
I would better use `float('Inf')` instead of `maxsize`

Reply:
Yeah, good suggestion